src/com/lftechnology/java/training/dipak/employeemanagement/controller/EmployeeController.java

```diff
@@ -0,0 +1,110 @@
+
+package com.lftechnology.java.training.dipak.employeemanagement.controller;
+
+import java.sql.ResultSet;
+import java.util.logging.Logger;
+
+import com.lftechnology.java.training.dipak.employeemanagement.domain.Employee;
+import com.lftechnology.java.training.dipak.employeemanagement.domain.UserType;
+import com.lftechnology.java.training.dipak.employeemanagement.service.EmployeeService;
+import com.lftechnology.java.training.dipak.employeemanagement.service.ServiceFactory;
+
+/**
+ * <p>
+ * This class is used to take the data from the user based on the various functionalities provided by its member function.
+ * </p>
+ * 
+ * @author Dipak Thapa<dipakthapa@lftechnology.com>
+ */
+public class EmployeeController {
+    private static Logger LOGGER = Logger.getLogger(EmployeeController.class.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make the "LOGGER" logger private static final. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1312)